app/src/main/java/org/literacyapp/battery/BatteryActivity.java

```diff
@@ -2,37 +2,18 @@
 
 import android.app.Activity;
 import android.content.Intent;
-import android.content.IntentFilter;
-import android.os.BatteryManager;
 import android.os.Bundle;
-import android.util.Log;
-import android.widget.TextView;
 
 
 public class BatteryActivity extends Activity {
```

Comment:
Rename to `BatteryLowActivity` to indicate a little better the purpose of the activity.